.gitignore

```diff
@@ -12,7 +12,6 @@ mod[s]/
 
 # Generated parsers
 *_generated.hpp
-*/local_user_settings.hpp
```

Comment:
local_user_settings is excluded on purpose, and I would like to keep it that way

Reply:
causes compile errors for the CI

Reply:
Specifically, if it is in the repo, then changes anyone makes to set their personal game directory will be pushed to everyone. Likewise, pulling will overwrite your personal game directory

Reply:
Ja, issue is that the CI needs the file for compiling, as its defines are used in the user_settings struct, it could probably be relocated, although not sure where

Reply:
Suggest putting the defines into a different pr then until we can come up with a solution, because obviously we can't put them in local settings then

Reply:
aight, ill just move them into src/special_define.hpp and adjust the contributing.md then

Reply:
It will lead to the same problem. If you expect people to edit these, their changes will constantly get sent to the repo. Is that what you want?

Reply:
No, thats why it was in local_user_settings.hpp, since people should change them, but these defines are needed for compile time (For CI), so it'll have to do

Reply:
OH wait nevermind, we can just exclude the .gitignore for local_user_settings for a single commit then add it back later, that'll make sure no ones copy gets pushed but the repo will maintain a valid version

Reply:
So, when I edit them, it is going to change your file the next time you pull ...

Reply:
your stuff would still abide by the .gitignore, afterall its done at pushing time ?

Reply:
oh, though other option is to add a file creation as part of the github action, so it'll create a local_user_settings.hpp file with the defines needed